packages/design-system/src/components/web-components/utils.test.ts

```diff
@@ -0,0 +1,28 @@
+import { isPossibleValue } from './utils';
+
+type Fruit = 'apple' | 'banana' | 'cherry';
+
+const acceptableFruit: Fruit[] = ['apple', 'banana', 'cherry'] as const;
```

Comment:
I added the Fruit type and accpetableFruit array to get closer to how we use this function in the wild. I'm currently researching whether there are type testing approaches or frameworks that might be useful, since I can't really test the types. I've found this [helpful article](https://www.totaltypescript.com/how-to-test-your-types) which suggests two separate libraries for testing types, and a unique approach using Expect and Equal. The problem with the Expect and Equal approach is that the 'test' is just checking if the code compiles. Which is correct, but kinda outside of how we do tests normally.

Reply:
[TIL](https://stackoverflow.com/questions/62028427/how-to-use-ts-type-as-a-value) that "You can't make a value from a type, but you can infer a type from a value."

I'm personally ok with a test that gets us closer to how it's used in the rest of the codebase.